docs/source/distributions.rst

```diff
@@ -210,6 +210,13 @@ Rejector
     :undoc-members:
     :show-inheritance:
 
+RobustHMM
```

Comment:
Can we rename this to `StudentTHMM` or something more specific, so as to leave room for alternative robust time series models like `StableHMM`?

Reply:
Or maybe `GammaGaussianHMM`, to distinguish it from a collection of independent `StudentT` distributions.

Reply:
done